Address/index.ts

```diff
@@ -0,0 +1,17 @@
+import { isly } from "isly"
+import { Alpha2 } from "../CountryCode"
```

Comment:
Alpha2 is never used

Reply:
`Alpha2` is the one we use the most. `Alpha3` and `Numeric` mostly only when integrating other peoples older APIs. Still useful.